src/markdown/ciceromark@0.3.0.cto

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace org.accordproject.ciceromark
+
+import org.accordproject.commonmark.Node from https://models.accordproject.org/markdown/commonmark@0.2.0.cto
+import org.accordproject.commonmark.Child from https://models.accordproject.org/markdown/commonmark@0.2.0.cto
+
+/**
+ * A model for Accord Project extensions to commonmark
+ */
+
+abstract concept Element extends Child {
+  o String name
+  o String elementType optional
+}
+
+concept Variable extends Element {
+  o String value
+  o String identifiedBy optional
+}
+
+concept FormattedVariable extends Variable {
+  o String format
+}
+
+concept EnumVariable extends Variable {
+  o String[] enumValues
+}
+
+concept Formula extends Element {
+  o String value
+  o String[] dependencies optional
+  o String code optional
+}
+
+abstract concept Block extends Element {
+}
+
+concept Clause extends Block {
+  o String src optional
+}
+
+concept Conditional extends Block {
+  o Boolean isTrue
+  o Node[] whenTrue
+  o Node[] whenFalse
+}
+
+concept Optional extends Block {
+  o Boolean hasSome
+  o Node[] whenSome
+  o Node[] whenNone
+}
+
+concept ListBlock extends Block {
+    o String type
```

Comment:
Enum for the type? Could be shared with CommonMark?

Reply:
This one comes out of the commonmark parser which gives you a string which is either `"bullet"` or `"ordered"`. Not sure it's worth mingling with that?